packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -694,14 +694,16 @@ - (void)invalidateLayer
     layer.cornerRadius = 0;
 
     RCTBorderColors borderColors = RCTCreateRCTBorderColorsFromBorderColors(borderMetrics.borderColors);
-
+    UIColor *transparentColor = [UIColor clearColor];
+    CGColorRef transparentBackgroundColor = [transparentColor resolvedColorWithTraitCollection:self.traitCollection].CGColor;
+          
     UIImage *image = RCTGetBorderImage(
         RCTBorderStyleFromBorderStyle(borderMetrics.borderStyles.left),
         layer.bounds.size,
         RCTCornerRadiiFromBorderRadii(borderMetrics.borderRadii),
         RCTUIEdgeInsetsFromEdgeInsets(borderMetrics.borderWidths),
         borderColors,
-        backgroundColor,
+        transparentBackgroundColor,
```

Comment:
I am not sure if we can get rid of the background here. If we set the background in the `layer` instead of `borderLayer`, components that have `clipsToBounds = false` may not be drawn correctly. This causes the above-mentioned bug of not clipping the background.